src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -343,7 +337,7 @@ export const SearchBarDropdown = ({
   ])
 
   return (
-    <Box className={isPhase1 ? styles.searchBarDropdownNft : styles.searchBarDropdown}>
```

Comment:
can you remove this no longer used style?

Reply:
yes